### DISCUSSION TITLE
show filter values in chart title or annotation layer

### DISCUSSION THREAD

**Discussion by n-berezina-nn** [apache/superset#25705] [Q&A / Help]
Hi!
My users request to show some filter values in chart title or in annotation layer, so they could export it to image and have needed information in one place.
I've been googling for a while but failed to find a way to achieve it.
I'll appreciate any help on this!
I am using superset 3.0.0

**Comment by aimmon** [comment]
I also have this type of development requirement, but the official response is that it is not supported

**Comment by dsKanishk** [comment]
Explore jinja templating.

**Reply by waqar-ahmed-malik** [reply]
Any basic idea to start on it?
Thanks in advance!

**Reply by n-berezina-nn** [reply]
I've tried jinja templates, they do not work in chart titles.

**Reply by dsKanishk** [reply]
Although it does not work in Chart titles,
We still can use jinja to retrieve values like filtered values, date selection, user logged in etc. and show them in table and remove borders using CSS and it can be visible as independent values which is the requirement in the first place so that it's a part of the viz.
